x/gateway/types/message_stake_gateway.go

```diff
@@ -42,7 +43,7 @@ func (msg *MsgStakeGateway) ValidateBasic() error {
 	// Validate the address
 	_, err := sdk.AccAddressFromBech32(msg.Address)
 	if err != nil {
-		return errors.Wrapf(ErrGatewayInvalidAddress, "invalid gateway address %s; (%v)", msg.Address, err)
+		return sdkerrors.Wrapf(ErrGatewayInvalidAddress, "invalid gateway address %s; (%v)", msg.Address, err)
```

Comment:
Here we are double importing `cosmossdk.io/errors` I am not aliasing so this doesnt change anything except adding a double import see: https://github.com/pokt-network/poktroll/pull/72/files#r1364287010 for a NIT around import naming convenction on this package I think we should come to aggreement on